src/view/viewTabs.component.tsx

```diff
@@ -102,7 +71,7 @@ function ViewTabs() {
               height: 'calc(100% - 48px)',
             }}
           >
-            <TabPanel value={value} label={value}>
+            <TabPanel<TabValue> value={value} label={value}>
```

Comment:
```suggestion
            <TabPanel value={value} label={value}>
```
It will automatically handle the type because it already has the type from the type of value. I found it a bit confusing at first and couldn't find examples of anyone else using templates like that in jsx. Same for the others.

Reply:
I did this for consistency. i know I didn't need to put it here but I've used if for the other ones

Reply:
Could you just remove them then? I struggled to find anyone else using it and chatgpt suggested its best practice to leave it out for this sort of case.

Reply:
I add it for consistency within our codebase as this is how it is use in other places. This makes it easy to maintain in the future

Reply:
Although now I actually have the proper name, I can find others using it was just that it was relatively new.

Reply:
I also just realised there are other places we don’t specify the generic types in JSX (not that I want us to 😄 ) e.g. <MaterialReactTable table={table} />, a lot of MUI components ListItemText, TextField etc, nobody uses it though because there is no need.